libnavigation-core/src/main/java/com/mapbox/navigation/core/trip/session/MapboxTripSession.kt

```diff
@@ -492,12 +483,12 @@ internal class MapboxTripSession(
         }
 
         unconditionalStatusPollingJob = ioJobController.scope.launch {
-            delay(UNCONDITIONAL_STATUS_POLLING_PATIENCE)
+            delay(locationPollingOptions.navigatorPatienceMillis)
```

Comment:
I took this for a test drive with ReplayActivity and it gets a little stuck. The simulated locations assume 1 second intervals, but it seems like something we would see

```
mapboxNavigation?.startTripSession(LocationPollingOptions.Builder()
                .navigatorPatienceMillis(100)
                .navigatorIntervalMillis(50)
                .locationEngineRequest(LocationEngineRequest.Builder(50L)
                    .setFastestInterval(50L)
                    .setPriority(LocationEngineRequest.PRIORITY_HIGH_ACCURACY)
                    .build())
                .build())
```

| replay route high frequency | replay history normal | replay history high frequency |
| -- | -- | -- |
| ![output](https://user-images.githubusercontent.com/3021882/99607383-8ebad280-29c0-11eb-9c25-d25a74cbf143.gif) | ![output](https://user-images.githubusercontent.com/3021882/99607528-d3df0480-29c0-11eb-80e5-df266fda2d5f.gif) | ![output](https://user-images.githubusercontent.com/3021882/99607764-4e0f8900-29c1-11eb-918d-a6cf64ad42cf.gif) |

Reply:
I tried changing to `.navigatorPredictionMillis(0L)` but you get the same choppy result

Reply:
Here's another ReplayHistory but with a slower interval. Less choppy, as expected. 

```
            mapboxNavigation?.startTripSession(LocationPollingOptions.Builder()
                .navigatorPatienceMillis(100)
                .navigatorIntervalMillis(50)
                .locationEngineRequest(LocationEngineRequest.Builder(50L)
                    .setFastestInterval(50L)
                    .setPriority(LocationEngineRequest.PRIORITY_HIGH_ACCURACY)
                    .build())
                .build())
```
![output](https://user-images.githubusercontent.com/3021882/99608411-a72bec80-29c2-11eb-8313-266ab12ade8f.gif)